src/app/containers/ATIAnalytics/beacon/index.js

```diff
@@ -2,26 +2,31 @@ import sendBeacon from '../../../lib/analyticsUtils/sendBeacon';
 import { buildATIEventTrackUrl } from '../atiUrl';
 
 export const sendEventBeacon = async ({
+  campaignName,
```

Comment:
Can the docs be updated here to reflect this refactor? https://github.com/bbc/simorgh/blob/be9497217bb82e2b1c80e7b7f7d962f5ba901a0a/src/app/containers/ATIAnalytics/README.md

Reply:
I think updating these docs is going to be a joint effort from Harry and myself after we combine the 2 hooks.

Reply:
Sounds good 👍